src/_css/img/moon.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" class="dark-mode-toggle-icon" viewBox="0 0 24 24"><path d="M20 8.69V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69zM12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6c3.31 0 6 2.69 6 6s-2.69 6-6 6z"/><path d="M0 0h24v24H0V0z" fill="none"/></svg>
```

Comment:
Can these files be minified further without breaking anything? E.g. the `class="dark-mode-toggle-icon"` _looks_ like it could be removed (but I’m probably missing something).

Reply:
Good catch, https://github.com/v8/v8.dev/pull/254/commits/4ae061636329bd0184ddd64fc6b878d783dbfe57 fixes this. I have run them through [svgomg](https://jakearchibald.github.io/svgomg/), but somehow the classes remained. Seems like https://github.com/svg/svgo/issues/1068 is dealing with this.